.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+      branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (Erlang/OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+    runs-on: ubuntu-latest
+    container: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.otp }}-alpine-3.11.6
```

Comment:
Just curious, is there any difference in performance between using this and using [actions/setup-elixir](https://github.com/actions/setup-elixir)? This runs a sibling container while the other runs everything directly in the host.

cc @ericmj

Reply:
Sorry, missed this one.

Yes, that would be the difference. Using `actions/setup-elixir` you add elixir to ubuntu's default image that has a bunch of stuff already installed. Using the container in this script will use a minimal alpine image with only elixir and erlang installed.

Reply:
@ericmj no worries. Which one would you recommend using?

Reply:
I am not sure. The actions are composable unlike the containers but if you only need elixir then the containers should be enough. It seems like GitHub generally favors using actions but in this case I would probably just go with what runs faster (I haven't run any benchmarks).

Reply:
Some informal benchmarking here: https://github.com/adrianomitre/elixir_github_actions_ci_playground/pull/3. It seems that `actions/setup-elixir` is marginally faster.

I would add that sibling containers with immutable images such as `hexpm/elixir` offer a more locally reproducible environment. Contrast the simplicity of `docker run -it -v $(pwd):/project hexpm/elixir:1.10.4-erlang-23.0.2-alpine-3.11.6 sh` with trying to setup a [GitHub Actions virtual environment](https://github.com/actions/virtual-environments) locally.

Reply:
👌 Thanks!